modules/op-scoping-github-token.adoc

```diff
@@ -0,0 +1,110 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: PROCEDURE
+[id="scoping-github-token_{context}"]
+= Scoping the GitHub token to additional repositories
+
+{pac} uses the GitHub App to generate a GitHub access token. {pac} uses this token to retrieve the pipeline payload from the repository and to enable the CI/CD processes to interact with GitHub repositories.
```

Comment:
```suggestion
{pac} uses the GitHub app to generate a GitHub access token. This token is used to retrieve the pipeline payload from the repository, and to enable the CI/CD processes to interact with GitHub repositories.
```
Unless the literal name is "Github App" I'd make app lowercase here.
Some suggestions re comma and wording, not crucial though.

Reply:
Can't agree with wording suggestions sorry, but doing the lowercase for app here.